### PR TITLE
Replace summary reports from result list

### PR DESCRIPTION
**Pull request by chiatt**
Replaces summary reports in 'add to set' search result list with displayname and displaydescription, re #661

### CODE REVIEW

File: afs/media/js/views/components/workflows/create-project-workflow/add-things-step.js [hunk 1]
```diff
@@ -386,38 +377,21 @@ define([
...
-                                datatypes: reportData[graphId].graph.datatypes
-                            }));
+                            source._source.tiles.forEach((tile) => {
+                                if (tile.data['22c15cfa-b498-11e9-b5e3-a4d18cec433a'] && 
+                                    tile.data['22c15cfa-b498-11e9-b5e3-a4d18cec433a'].length &&
+                                    tile.data['22c15cfa-b498-11e9-b5e3-a4d18cec433a'][0] === '26094e9c-2702-4963-adee-19ad118f0f5a') {
+                                    source.identifier = tile.data['22c169b5-b498-11e9-bdad-a4d18cec433a'];
+                                }
+                            });
+                            console.log(source);
```
Comment (njkim) [inline afs/media/js/views/components/workflows/create-project-workflow/add-things-step.js:387]: console.log may not be necessary here?

File: afs/templates/views/components/workflows/create-project-workflow/add-things-step.htm [hunk 2]
```diff
@@ -57,7 +60,7 @@
                         <div class="wf-multi-tile-card-info-details">
                         <dt data-bind="text: $data.displayname"></dt>
                         <dd>
-                            <a data-bind="text: $data.displaydescription"></a>
+                            <a data-bind="text: $($data.displaydescription).text() || 'No description'"></a>
```
Comment (njkim) [inline afs/templates/views/components/workflows/create-project-workflow/add-things-step.htm:63]: This doesn't seem to be working, almost always return "No description"

File: afs/media/js/views/components/workflows/create-project-workflow/add-things-step.js (hunk 1 above)
Comment (chiatt) [reply afs/media/js/views/components/workflows/create-project-workflow/add-things-step.js]: Ah - thanks. I've removed it.

File: afs/templates/views/components/workflows/create-project-workflow/add-things-step.htm (hunk 2 above)
Comment (chiatt) [reply afs/templates/views/components/workflows/create-project-workflow/add-things-step.htm:63]: Are you sure that your objects have a description statement? It seems to be working okay for me:

image

Comment (njkim) [reply afs/templates/views/components/workflows/create-project-workflow/add-things-step.htm:63]: Screen Shot 2021-10-01 at 4 37 58 PM

Comment (njkim) [reply afs/templates/views/components/workflows/create-project-workflow/add-things-step.htm:63]: $() may not be working on a simple string.

Comment (chiatt) [reply afs/templates/views/components/workflows/create-project-workflow/add-things-step.htm]: I replaced the jQuery text method for the ko html binding. It should work for you now.

Comment (njkim) [reply afs/templates/views/components/workflows/create-project-workflow/add-things-step.htm]: yep, html binding works.

**Pull request merged**